packages/web/.eslintrc.js

```diff
@@ -41,12 +41,6 @@ module.exports = {
     // allow reassign in properties
     'no-param-reassign': ['warn', { props: false }],
     // support monorepos
-    'import/no-extraneous-dependencies': [
-      'error',
-      {
-        packageDir: ['./', '../../'],
-        peerDependencies: true,
```

Comment:
This is also the `sass` thing, isn't it…

Reply:
this was removed in previous change